src/app/form/(formSteps)/disclosures/page.tsx

```diff
@@ -1,9 +1,89 @@
 "use client";
 
+import { setKeyValue } from "@/app/form/_utils/sessionStorage";
+import FormProgressButtons from "@form/(formSteps)/components/FormProgressButtons";
+import { routeToNextStep, useFormProgressPosition } from "@form/_utils/formProgressRouting";
+import { Fieldset, Form, Radio } from "@trussworks/react-uswds";
+import { useRouter } from "next/navigation";
 import React from "react";
+import { type SubmitHandler, useForm } from "react-hook-form";
 
-const FormStep: React.FC = () => {
-  return <div>hi</div>;
+interface DisclosuresData {
+  hadBreakfast: "true" | "false" | "";
+  breakfastWasTasty: "true" | "false" | "";
+}
+
+const DisclosuresStep: React.FC = () => {
+  const router = useRouter();
+  const formProgressPosition = useFormProgressPosition();
+  const { register, handleSubmit, watch } = useForm<DisclosuresData>({
+    defaultValues: {
+      hadBreakfast: "",
+      breakfastWasTasty: "",
+    },
+  });
+  const hadBreakfast = watch("hadBreakfast");
+  const breakfastWasTasty = watch("breakfastWasTasty");
+
+  const onSubmit: SubmitHandler<DisclosuresData> = (data) => {
+    let key: keyof DisclosuresData;
+    for (key in data) {
+      const value = data[key] ?? "";
+      setKeyValue(key, value);
+    }
+    routeToNextStep(router, formProgressPosition);
+  };
+
+  return (
+    <div>
+      <Form onSubmit={handleSubmit(onSubmit)} className="maxw-full">
+        <div className="maxw-tablet">
+          <h2 className="font-heading-md">Breakfast information</h2>
+          <p>Have you had breakfast today?</p>
+          <Fieldset legend={<p className="font-ui-xs">Select one</p>} legendStyle="default">
```

Comment:
PEBBLE - the screen reader starts at the top of the form on this page. This is not necessarily an incorrect assumption, but want to flag this behavior and make sure it is intentional